lib/cache/throttlingCache.ts

```diff
@@ -0,0 +1,65 @@
+"use server";
+import { logMessage } from "@lib/logger";
+import { getRedisInstance } from "../integration/redisConnector";
+import { getWeeksInSeconds } from "@lib/utils/date/dateConversions";
+
+export const getThrottling = async (
+  formId: string
+): Promise<{ rate: string | null; expires: number }> => {
+  const getParameter = `rate-limit:${formId}`;
```

Comment:
You could declare the key prefix as a const variable at the top of the file and reuse it across all your functions. This way, if we ever need to change it, there will only one line of code to update.

Example: https://github.com/cds-snc/forms-api/blob/main/src/lib/rateLimiting/tokenBucketProvider.ts#L14